### PR TITLE
Fix #189: updates to frontend for mobile

### LINKED ISSUE #189: Mobile issues
With the new Html designed we have a few issues with font size on mobile menu, and on mobile  there is mild issues with area of access in iphones and  small phones, so fixing that issue

### PR DESCRIPTION
**Pull request by cagomezr**
Fixing issue #189

On mobile views we have the following issues:

- The buttons when tested with  poor vision individuals  I got feed back that they should be  sligthly larger so I updated the  font size, by a 40% ratio.

- On  our close functionality we have an issue with the system where  when we close on  devices of small range and strange screen targets ( Asus Zenfone 2 and  iPhone 4 tested)   the close menu  function didn't  react , the solution was to increasing the   close  "x"  to a rate of  75%

 this changes can be seen prototype in

https://cagomezr.github.io/telescope-demo/

### CODE REVIEW

File: src/frontend/style/main.css [hunk 1]
```diff
@@ -227,8 +227,13 @@ article section p {
         position: absolute;
         right: 20px;
         top: 20px;
+        font-size:1.75em;
     }
-
+    
+	.nav-button{
+		font-size: 1.25em;
```
Comment (humphd) [inline src/frontend/style/main.css]: This needs to get added to the block on line 76 above with the rest of the class rules for `.nav-button`

File: src/frontend/style/main.css [hunk 2]
```diff
@@ -227,8 +227,13 @@ article section p {
         position: absolute;
         right: 20px;
         top: 20px;
+        font-size:1.75em;
```
Comment (humphd) [inline src/frontend/style/main.css]: Space after your `:` please.

**Comment by humphd** [comment]
You have more lint issues in here:

```
src/frontend/style/main.css
 90:5  ✖  Unexpected duplicate        declaration-block-no-duplicate-properties
          "font-size"
```

**Comment by ODAVING** [comment]
Can you please squash the commits once all the changes are done? 
Thank you!

**Comment by humphd** [comment]
Your PR now does nothing (0 files changed).

**Comment by cagomezr** [comment]
Im having  issues rebasing my code.  it  seems

**Comment by humphd** [comment]
@cagomezr no problem.  Can I suggest you close this, and do a new branch off of `master` and do your change again, since it's small.  It will be faster than trying to fix this history.

**Pull request closed without merging**